app/src/main/java/nl/tudelft/cs4160/trustchain_android/main/OverviewConnectionsActivity.java

```diff
@@ -127,6 +146,39 @@ private void initKey() {
         }
     }
 
+    /**
```

Comment:
Might be better to place this function in a different/new class

Reply:
I moved it to UserConfigurationActivity, which is a better place since that is the first activity that get's called. Moving it to a new class is not necessary I think. It needs the context from activities anyway.